website/docs/e2e.md

```diff
@@ -96,3 +96,4 @@ Via CLI:
 - [e2e-ci-example-gh-actions](https://github.com/juanpicado/e2e-ci-example-gh-actions)
 - [verdaccio-end-to-end-tests](https://github.com/juanpicado/verdaccio-end-to-end-tests)
 - [verdaccio-fork](https://github.com/juanpicado/verdaccio-fork)
+- [simplified e2e testing using verdaccio](https://github.com/rluvaton/e2e-verdaccio-example)
```

Comment:
@rluvaton ahhh, you updated main docs, which is deployed at 6.x https://verdaccio.org/docs/next/e2e for 5.x you need to update same file in the versioned folder.

Reply:
Thanks, created a new pr